driver-core/src/main/com/mongodb/internal/capi/MongoCryptHelper.java

```diff
@@ -26,8 +31,9 @@
 import java.util.ArrayList;
 import java.util.List;
 import java.util.Map;
+import java.util.concurrent.TimeUnit;
 
-public final class MongoCryptOptionsHelper {
+public final class MongoCryptHelper {
```

Comment:
Generalized the name so that the refactored method makes sense here.

Reply:
Ack